services/wallet/collectibles/commands.go

```diff
@@ -114,24 +137,32 @@ func (c *periodicRefreshOwnedCollectiblesCommand) loadOwnedCollectibles(ctx cont
 // Fetches owned collectibles for a ChainID+OwnerAddress combination in chunks
 // and updates the ownershipDB when all chunks are loaded
 type loadOwnedCollectiblesCommand struct {
-	chainID     walletCommon.ChainID
-	account     common.Address
-	manager     *Manager
-	ownershipDB *OwnershipDB
-	walletFeed  *event.Feed
+	chainID                walletCommon.ChainID
+	account                common.Address
+	manager                *Manager
+	ownershipDB            *OwnershipDB
+	walletFeed             *event.Feed
+	receivedCollectiblesCh chan<- OwnedCollectibles
 
 	// Not to be set by the caller
 	partialOwnership []thirdparty.CollectibleUniqueID
 	err              error
 }
 
-func newLoadOwnedCollectiblesCommand(manager *Manager, ownershipDB *OwnershipDB, walletFeed *event.Feed, chainID walletCommon.ChainID, account common.Address) *loadOwnedCollectiblesCommand {
+func newLoadOwnedCollectiblesCommand(
```

Comment:
I'm a bit puzzled how
```
command := newLoadOwnedCollectiblesCommand(c.manager, c.ownershipDB, c.walletFeed, c.chainID, c.account, receivedCollectiblesCh)
```
is better than
```
command := &loadOwnedCollectiblesCommand{c.manager, c.ownershipDB, c.walletFeed, c.chainID, c.account, receivedCollectiblesCh}
```

and why we actually need an extra `newSomething` method in case if there is nothing except passing fields to the structure. But well if that's an idiomatic way to do things then that's just my problem. I also know it was not introduced in this specific PR, just that those lines have to be changed because of this pattern.

Reply:
ok I see there are fields which
```
// Not to be set by the caller
```
still named fields could be used. Anyways though, that's likely just my problem :D

Reply:
> to the struct and its initialization are limited to one spot

yep I get it and agree, just seeing lots of functions with 3+ arguments make me uncomfortable, but again my problem